lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -10,6 +10,18 @@
   }
 }
 
+.card__column {
+  .photo__actions, .caption {
```

Comment:
We shouldn't be nesting a `.photo__action` element inside a `.card`. Either this should be a new child element of the `.card`, or we should see if `.photo` makes sense to use for the markup for this interface as well.